setup.py

```diff
@@ -516,6 +516,7 @@ def add_gcc_paths(self):
             os.makedirs(self.build_temp)
         ret = os.system('%s -E -v - </dev/null 2>%s 1>/dev/null' % (gcc, tmpfile))
         is_gcc = False
```

Comment:
The intent of this function is were unclear from the function name. I suggest to rename it to "def add_cross_compilation_paths()". So it would be less surprising to see this function supporting clang as well as gcc :-)

Please also rename "gcc" variable to "cc".

Reply:
Which one is better, add_cross_compilation_paths or add_cross_compiling_paths? Everywhere is using cross_compiling but no cross_compilation used in codes.

Reply:
Oh sorry: "add_cross_compiling_paths" is more consistent with the existing "cross_compiling" global variable ;-)